integration_tests/snaps/rust-with-revision/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "rust-with-revision"
+version = "0.1.0"
+authors = ["Marius Gripsgard <mariogrip@ubuntu.com>"]
+
+[dependencies]
+log = "*"
+rustc_version = "= 0.1.2"
+rustc_version_runtime = "0.1.0"
```

Comment:
ouch, they fixed the bug I reported a lot faster than what I expected:
https://github.com/seppo0010/rustc-version-runtime-rs/issues/1

From the few bits I learned from cargo this morning, I think that will make it break.
You should change the two lines above for this one:
rustc_version_runtime = "0.1.3"